mteb/tasks/Retrieval/XMarketESRetrieval.py

```diff
@@ -0,0 +1,49 @@
+import datasets
+
+from ...abstasks.AbsTaskRetrieval import AbsTaskRetrieval
+from ...abstasks.BeIRTask import BeIRTask
+
+
+class XMarketES(AbsTaskRetrieval):
+    _EVAL_SPLIT = 'test'
+
+    @property
+    def description(self):
+        return {
+            "name": "XMarketES",
+            "hf_hub_name": "jinaai/xmarket_ml",
+            "description": "XMarket is an ecommerce category to product retrieval dataset in Spanish.",
+            "reference": "https://xmrec.github.io/",
+            "type": "Retrieval",
+            "category": "s2p",
+            "eval_splits": ["test"],
+            "eval_langs": ["es"],
+            "main_score": "ndcg_at_10",
+            "revision": "705db869e8107dfe6e34b832af90446e77d813e3",
+        }
+
+    def load_data(self, **kwargs):
+        if self.data_loaded:
+            return
+
+        query_rows = datasets.load_dataset(
+            self.description["hf_hub_name"],
+            "queries",
+            languages=['es'],
+            split=self._EVAL_SPLIT,
+            trust_remote_code=True,
+        )
+        corpus_rows = datasets.load_dataset(
+            self.description["hf_hub_name"], "corpus", languages=['es'], split=self._EVAL_SPLIT, trust_remote_code=True
```

Comment:
maybe it would be nicer to replace languages=['es'] with languages=kwargs['eval_langs']

Reply:
or self.description['eval_langs']

not sure which is correct

Reply:
I think eval_langs is supposed to be the set of supported languages. There is probably another kwargs in in load_data for the languages. I can look into it